devel/README.md

```diff
@@ -0,0 +1,64 @@
+# e3e - e2e + 1
+
+This directory contains tools and scripts used to create development and
+testing environments for cert-manager.
+
+You can invoke different parts of e3e by running the various scripts throughout
+the repository.
+
+## Tool dependencies
+
+The scripts in this directory commonly require additional tooling, such as
+access to `kubectl`, `helm`, `kind` and a bunch of other things.
+
+If you already have these tools available on your host system, the scripts
+should just work, so long as the versions you have installed are roughly
+compatible.
+
+If you are running into issues with your host-installed tools, Bazel provides
+versioned access to all of the required tools for the e3e scripts.
+
+To setup your shell to use the Bazel provided versions of these tools, run the
+following from the **root of the repository**:
+
+```
+$ export PATH="$(pwd)/test/e3e/bin:$PATH"
```

Comment:
I need to go through and remove references to 'e3e'

Reply:
Is there a way to add this into the makefile?

Reply:
I've updated this to make it the default 😄